app/__init__.py

```diff
@@ -31,14 +39,15 @@ def generate_sample_results(limit):
 @lsns.route('/')
 class LogsSearch(Resource):
     @lsns.expect(search_parser)
-    def get(self):
+    def post(self):
         args = search_parser.parse_args()
 
-        log_filename = 'generated.log'
-        result_id = searchIn(log_filename)
+        #log_filename = 'generated.log'
+        #result_id = searchIn(log_filename)
```

Comment:
result_id is important and it need to be in response, because it is id given to front. Then front uses it to get results found by spark

Reply:
Done